### PR TITLE
Add windows CUDA 12.6 builds

### PR DESCRIPTION
**Pull request by atalman**
Enables windows CUDA 12.6 builds. Since these where enabled on the core

### CODE REVIEW

**Comment by clee2000** [comment]
Do you mind rerunning `python -m  tools.tests.test_generate_binary_build_matrix --update-reference-files`?  It doesn't matter much, I think most of the test file changes are just due to a change in the indentation

File: tools/scripts/generate_binary_build_matrix.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
 import os
 import sys
 
-from typing import Dict, List, Optional, Tuple, Any, Callable
+from typing import Any, Callable, Dict, List, Optional, Tuple
```
Comment (malfet) [inline tools/scripts/generate_binary_build_matrix.py:20]: Why this change is needed?

File: tools/scripts/generate_binary_build_matrix.py [hunk 2]
```diff
@@ -148,10 +148,10 @@ def initialize_globals(channel: str, build_python_only: bool) -> None:
     else:
         PYTHON_ARCHES = PYTHON_ARCHES_DICT[channel]
     WHEEL_CONTAINER_IMAGES = {
-       "11.8": "pytorch/manylinux-builder:cuda11.8",
-       "12.1": "pytorch/manylinux-builder:cuda12.1",
-       "12.4": "pytorch/manylinux-builder:cuda12.4",
-       "12.6": "pytorch/manylinux2_28-builder:cuda12.6",
+        "11.8": "pytorch/manylinux-builder:cuda11.8",
+        "12.1": "pytorch/manylinux-builder:cuda12.1",
```
[the remark this answers is not in the document]
Comment (malfet) [reply tools/scripts/generate_binary_build_matrix.py:152]: Plus one on the above..

**Review by malfet — COMMENTED** [review]
Looks like PR contains number of unrelated changes. More specifically, it feels weird to see `test/assets` updates for architectures that has nothing to do with the change

File: tools/scripts/generate_binary_build_matrix.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (atalman) [reply tools/scripts/generate_binary_build_matrix.py:152]: Its needed if you want to compile against torch release: https://github.com/pytorch/ao/actions/runs/12123278267

File: tools/scripts/generate_binary_build_matrix.py (hunk 1 above)
Comment (atalman) [reply tools/scripts/generate_binary_build_matrix.py]: Disabled auto-formatted in vscode. Sorry about this

**Review by huydhn — APPROVED** [review]

**Review by zxiiro — APPROVED** [review]
